src/status_im/utils/test.cljs

```diff
@@ -38,9 +38,9 @@
 
     :getNodeConfig (fn [] (types/clj->json {:WakuV2Config ""}))
 
-    :backupDisabledDataDir (fn [] "./smth")
+    :backupDisabledDataDir (fn [] (str test-dir "/backup"))
```

Comment:
(it's backup disabled directory, so it would be `no_backup` :) , but it's all good)